crates/pathfinder/resources/block_hash.py

```diff
@@ -0,0 +1,53 @@
+from distutils.command.build import build
```

Comment:
I have a feeling this depends on the `py/.venv` environment, I think it would be better in there than under pathfinder? I think there's a contract hash related helper as well.

Reply:
Ah, no, we don't actually need that line. Must have been added by VS Code's Python extension and I didn't notice it...

Reply:
Ah, OK. Sure, we can just move it there.

Reply:
Moved there.